dev/ci/ci-equations.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+ci_dir="$(dirname "$0")"
+
+# This script could be included inside other ones
+# Let's avoid to source ci-common twice in this case
+if [ -z "${CI_BUILD_DIR}"];
```

Comment:
This sticks the `]` together with the string resulting in missing ] error.
I think instead of this per-script system we should make ci-common idempotent.
eg have it export HAS_CI_COMMON=true like we're writing C headers.
Idempotent ci-common is outside the scope of this PR but please fix the missing ] error by adding a space before the ].

Reply:
In fact this test should be get rid of, only one script is using it and indeed `ci-common.sh` was not designed to be included like that.

Reply:
I removed the test